lib/models/movie.dart

```diff
@@ -7,37 +9,79 @@ class Movie {
   final List<String> genres;
   final String pathToBackdropImg;
   final String pathToPosterImg;
+  final bool adult;
+  final String originalLanguage;
+  final int id;
+  final double popularity;
+  final bool video;
+  final int voteCount;
 
   String get score => ((voteAverage * 10).toInt().toString());
 
-  const Movie(
-    this.title,
-    this.originalTitle,
-    this.overview,
-    this.releaseDate,
-    this.voteAverage,
-    this.genres,
-    this.pathToBackdropImg,
-    this.pathToPosterImg,
-  );
+  const Movie({
+    required this.title,
+    required this.originalTitle,
+    required this.overview,
+    required this.releaseDate,
+    required this.voteAverage,
+    required this.genres,
+    required this.pathToBackdropImg,
+    required this.pathToPosterImg,
+    required this.adult,
+    required this.originalLanguage,
+    required this.id,
+    required this.popularity,
+    required this.video,
+    required this.voteCount,
+  });
 
   factory Movie.fromStatic() => const Movie(
-        'The Super Mario Bros. Movie',
-        'The Super Mario Bros. Movie',
-        'While working underground to fix a water main, Brooklyn '
+        title: 'The Super Mario Bros. Movie',
+        originalTitle: 'The Super Mario Bros. Movie',
+        overview: 'While working underground to fix a water main, Brooklyn '
             'plumbers—and brothers—Mario and Luigi are transported down a mysterious '
             'pipe and wander into a magical new world. But when the brothers are separated, '
             'Mario embarks on an epic quest to find Luigi.',
-        '2023-04-05',
-        7.8,
-        [
+        releaseDate: '2023-04-05',
+        voteAverage: 7.8,
+        genres: [
           'Animation',
           'Family',
           'Adventure',
           'Fantasy',
           'Comedy',
         ],
```

Comment:
With the new genres data that you have here instead to use the list of string, you can have a list of int, with the id's for the genres

Reply:
ended up making a new MovieModel class that works with IDs and maps them to a Movie (entity) class that references the actual Genre entity